gallery/src/components/NxList/NxListExamples.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxListSimpleExample from './examples/NxListSimple';
+import NxListClickableExample from './examples/NxListClickable';
+import NxListClickableLinkExample from './examples/NxListClickableLink';
+import NxListBulletedExample from './examples/NxListBulleted';
+import NxListActionsExample from './examples/NxListActions';
+import NxListMultiLineExample from './examples/NxListMultiLine';
+import NxListEmptyExample from './examples/NxListEmpty';
+import NxListErrorExample from './examples/NxListError';
+import NxListLoadingExample from './examples/NxListLoading';
+import NxListDescriptionExample from './examples/NxListDescription';
+import { NxCode } from '@sonatype/react-shared-components';
+
+const NxListSimpleExampleCode = require('./examples/NxListSimple?raw'),
+    NxListClickableExampleCode = require('./examples/NxListClickable?raw'),
+    NxListClickableLinkExampleCode = require('./examples/NxListClickableLink?raw'),
+    NxListBulletedExampleCode = require('./examples/NxListBulleted?raw'),
+    NxListActionsExampleCode = require('./examples/NxListactions?raw'),
```

Comment:
Module loading is case-insensitive on Mac but case-sensitive on Linux.  This line breaks the build for me.

```suggestion
    NxListActionsExampleCode = require('./examples/NxListActions?raw'),
```